lua/wikis/callofduty/FilterButtons/Config.lua

```diff
@@ -0,0 +1,43 @@
+---
+-- @Liquipedia
+-- wiki=callofduty
+-- page=Module:FilterButtons/Config
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Tier = require('Module:Tier/Utils')
+local Config = {}
+
+local Game = require('Module:Game')
+
+Config.categories = {
+	{
+		name = 'liquipediatier',
+		property = 'liquipediaTier',
+		load = function(category)
+			category.items = {}
+			for _, tier in Tier.iterate('tiers') do
+				table.insert(category.items, tier.value)
+			end
+		end,
+		defaultItems = {'1', '2', '3'},
+		transform = function(tier)
+			return Tier.toName(tier)
+		end,
+		expandKey = "game",
+	},
+	{
+		name = 'game',
+		property = 'game',
+		expandable = true,
+		load = function(category)
+			category.items = Game.listGames({ordered = true})
+		end,
```

Comment:
This could be an option for the limited selection of games.
```suggestion
		items = {'bo6', 'wz', 'codm'}
```
Another option would be to add a flag `active = true` to the games in module:info, and adjust the game module to be able to list only these.